db/re/skill_db.yml

```
@@ -42681,6 +42681,101 @@ Body:
     Description: The Vigilante At Night Gun Shotgun
     MaxLevel: 5
     CastCancel: true
+  - Id: 5500
+    Name: NW_WILD_SHOT
+    Description: Wild Shot
+    MaxLevel: 5
+    Type: Weapon
+    TargetType: Attack
+    DamageFlags:
+      Critical: true
+      Splash: true
+    Range: -9
+    Hit: Multi_Hit
+    HitCount: 7
+    Element: Weapon
+    SplashArea:
+      - Level: 1
+        Area: 2
+      - Level: 2
+        Area: 2
+      - Level: 3
+        Area: 2
+      - Level: 4
+        Area: 3
+      - Level: 5
+        Area: 3
+    GiveAp: 2
+    AfterCastActDelay: 1000
+    Cooldown: 500
+    CastCancel: true
+    FixedCastTime: 1000
+    Requires:
+      SpCost:
+        - Level: 1
+          Amount: 60
+        - Level: 2
+          Amount: 63
+        - Level: 3
+          Amount: 66
+        - Level: 4
+          Amount: 69
+        - Level: 5
+          Amount: 72
+      Weapon:
+        Rifle: true
+        Revolver: true
+      Ammo:
+        Bullet: true
+      AmmoAmount: 5
+  - Id: 5501
+    Name: NW_MIDNIGHT_FALLEN
+    Description: Midnight Fallen
+    MaxLevel: 5
+    Type: Weapon
+    TargetType: Ground
+    Range: -9
+    Hit: Multi_Hit
+    HitCount: 3
+    Element: Weapon
+    DamageFlags:
+      Splash: true
+      IgnoreDefense: true
+    SplashArea:
+      - Level: 1
+        Area: 2
+      - Level: 2
+        Area: 2
+      - Level: 3
+        Area: 2
+      - Level: 4
+        Area: 3
+      - Level: 5
+        Area: 3
+    GiveAp: 2
+    AfterCastActDelay: 1000
+    Cooldown: 500
+    CastCancel: true
+    FixedCastTime: 1000
```

```suggestion
    AfterCastActDelay: 800
    Cooldown: 700
    CastCancel: true
    FixedCastTime: 1000    # TODO
```
according to https://www.inven.co.kr/board/ro/1951/6360